tools/jenkins-slave-creation-unix/conf-ubuntu-gpu/infrastructure.tfvars

```diff
@@ -17,13 +17,14 @@
 
 key_name = "REDACTED"
 key_path = "~/.ssh/REDACTED"
-instance_type = "g3.8xlarge"
+instance_type = "g4dn.4xlarge"
 
 s3_config_bucket = "mxnet-ci-slave-dev"
-s3_config_filename = "ubuntu-gpu-g3-config.tar.bz2"
-slave_install_script  = "conf-ubuntu-gpu-g3/install.sh"
-shell_variables_file = "conf-ubuntu-gpu-g3/shell-variables.sh"
-ami = "ami-bd8f33c5" # ftp://64.50.236.216/pub/ubuntu-cloud-images/query/xenial/server/released.txt
-instance_name = "Slave-base_Ubuntu-GPU-G3"
+s3_config_filename = "ubuntu-gpu-g4-config.tar.bz2"
```

Comment:
You can get rid of `g4` here, as the script targets all gpu instances now

Reply:
addressed